CHANGELOG.md

```diff
@@ -5,9 +5,22 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [Unreleased]
+## [Unreleased](xxx) - 2021-02-04
 
+### Added
+
+- Replaced cassandra, postgres and kafka with upstream charts ( https://github.com/Telecominfraproject/wlan-cloud-helm/pull/49 )
+- Removed hardcoded docker secret in favor of variables ( https://github.com/Telecominfraproject/wlan-cloud-helm/pull/53 )
+
+### Replaced
```

Comment:
Replaced isn't an official type defined by keepachangelog. Would like to limit the types to [these](https://keepachangelog.com/en/1.0.0/#how).